examples/RayTune/h1st_tune.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os    
+
+from filelock import FileLock 
+from ray import tune
+
+from examples.RayTune.tensorflow_mlp_classifier import TensorflowMLPClassifier
+# kwargs = {
+#     'units': int(config['units']),
+#     'epochs': int(config['epochs']), 
+#     'lr': config['lr'], 
+#     'n_layer': int(config['n_layer'])
+# }
+model_class = TensorflowMLPClassifier
+
+class H1stTunable(tune.Trainable):    
+    def setup(self, config):
+        self.timestep = 0
+#         self.config = config
+#         self.timestep = 0
+        self.kwargs = {}
+        self.hyperparameter = getattr(
+            model_class, 'hyperparameter', {})['tuning_param']
+        for k, v in self.hyperparameter.items():
+            self.kwargs[k] = v(config[k])
+        self.h1_ml_model = model_class(**self.kwargs)
+
+        DATA_ROOT = './data/credit_card'
+        lock_file = f'{DATA_ROOT}/data.lock'
+        if not os.path.exists(DATA_ROOT):
+            os.makedirs(DATA_ROOT)
+        with FileLock(os.path.expanduser(lock_file)):
+            data = self.h1_ml_model.load_data()
+        self.prepared_data = self.h1_ml_model.prep(data)  
+        
+    def step(self):
+        self.timestep += 1
```

Comment:
[line#38-41] self.h1_ml_model is repeated. Looks quite busy. Should it be? @TgithubJ 

m = self.h1_ml_model
m.train(self.prepared_data)
m.evaluate(self.prepared_data)
acc = m.metrics['accuracy']
m.persist(str(self.timestep) + str())